instrumentation/logback/logback-mdc-1.0/library/src/test/java/io/opentelemetry/instrumentation/logback/mdc/v1_0/LogbackTest.java

```diff
@@ -18,4 +22,15 @@ class LogbackTest extends AbstractLogbackTest {
   public InstrumentationExtension getInstrumentationExtension() {
     return testing;
   }
+
+  @Test
+  void testMdcMutable() {
+    TestAppender testAppender = TestAppender.instance;
+    runWithSpanAndBaggage("test", baggage, () -> logger.info("log message"));
+
+    assertThat(testAppender.lastEvent.getMessage()).isEqualTo("log message");
+    Map<String, String> map = testAppender.lastEvent.getMDCPropertyMap();
```

Comment:
There are two instrumentation points: https://github.com/open-telemetry/opentelemetry-java-instrumentation/blob/8ff6cb0de04b1862bc1e04613583e4276520a9c2/instrumentation/logback/logback-mdc-1.0/javaagent/src/main/java/io/opentelemetry/javaagent/instrumentation/logback/mdc/v1_0/LoggingEventInstrumentation.java#L54 
Is there any reason to miss `getMdc` test here?

Reply:
This is a library instrumentation tests. It only serves as a regression test to ensure that we don't accidentally introduce something similar to the union map in the future.